assets/questions/palindrome.js

```diff
@@ -0,0 +1,159 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Balanced Palindrome.
+ */
+
+globalData.questions['palindrome'] = {  // eslint-disable-line dot-notation
+  title: 'Balanced Palindrome',
+  starterCode: {
+    python:
+`def isPalindrome(s):
+    return False
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def createPalindromeString(cls, chr_set, input_size):
```

Comment:
character_set

Also, I think we're using createPalindromeString-style casing, not create_palindrome_string, so make sure you're being consistent.